utils/zerovec/src/ule/mod.rs

```diff
@@ -0,0 +1,72 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! Traits over unaligned little-endian data (ULE, pronounced "yule").
+
+mod chars;
+mod plain;
+
+pub use chars::CharULE;
+pub use plain::PlainOldULE;
+
+/// Fixed-width, byte-aligned data that can be cast to and from a little-endian byte slice.
+pub trait ULE
```

Comment:
`request`: add example of `ULE` use on arbitrary struct.

Reply:
I'll think about how to do this.  It's not trivial to put a `ULE` example in the docs.  I wrote *chars.rs* as itself an example of how one might implement `ULE` for an arbitrary type.  Maybe an example file?